src/aitsi/m3spin/pkb/interfaces/Ast.java

```diff
@@ -13,6 +14,10 @@ public interface Ast {
     TNode createTNode(EntityType et) throws IllegalNodeTypeException;
 
     TNode setRoot(TNode node);
+    Parent getParentMethods();
```

Comment:
Podoba mi się też takie rozwiązanie, jakie zaproponowaliśmy z Szymonem kończąc Parser (gałąź `parser-tworzenie-ast`) - że używamy u siebie jednego interfejsu PKB, który wystawia wszystkie metody poszczególnych interfejsów PKB, ale w implementacji przekierowuje wywołania do implementacji poszczególnych interfejsów, które właśnie napisaliście 😃 Taki ogólny interfejs byłby fasadą, przekierowującą wywołania do poszczególnych implementacji. Jeszcze nie czuję, jakie podejście jest właściwe.
Powinniśmy to wspólnie przedyskutować 🗣️

Reply:
Im dłużej myślę, to jednak to rozwiązanie z fasadą zawierającą wszystkie metody wydaje mi się słabe, a wasze coraz bardziej lubię XD Ale i tak warto przedyskutować

Reply:
Moim zdaniem fasada wydaje się być dobrym pomysłem

Reply:
Do przedyskutowania